test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormTest.cs

```diff
@@ -487,6 +565,47 @@ public void BeginForm_EndForm_SuppressAntiforgeryToken()
                 writer.GetStringBuilder().ToString());
         }
 
+        [Theory]
+        [InlineData(FormMethod.Get)]
+        [InlineData(FormMethod.Post)]
+        public void BeginForm_EndForm_DoesNotSuppressAntiforgeryTokenWhenAntiforgeryIsTrue(FormMethod method)
```

Comment:
Up to you but `Begin[Route]Form_EndForm_MethodDoesNotSuppressAntiforgeryTokenWhenAntiforgeryIsTrue()` would be slightly more complete. Or perhaps `Begin[Route]Form_EndForm_AntiforgeryTrueOverridesMethod()`.